src/XIVLauncher.Core/Components/SettingsPage/Tabs/SettingsTabWine.cs

```diff
@@ -23,6 +23,17 @@ public SettingsTabWine()
                 CheckVisibility = () => startupTypeSetting.Value == WineStartupType.Command
             },
 
+            new SettingsEntry<bool>("Enable GameMode", "Enable launching with GameMode optimizations.", () => Program.Config.GameModeEnabled ?? true, b => Program.Config.GameModeEnabled = b)
+            {
+                CheckVisibility = () => File.Exists("/usr/lib/libgamemodeauto.so.0")
```

Comment:
Do you think it'd be better to use CheckValidity here, to let people know that they have to install this first?

Reply:
Not going to lie, forgot it existed. I do think it'd be a preferable option to hiding it all together so I'll check it in the morning.